RickAndMorty/Coordinator/Coordinator.swift

```diff
@@ -13,6 +13,8 @@ final class Coordinator {
     
     init(tabBarController: UITabBarController = UITabBarController()) {
         self.tabBarController = tabBarController
+        tabBarController.tabBar.tintColor = .systemGreen
```

Comment:
This luiine be moved to `setupTabBar`

Reply:
Also I don't think that green looks good
Provide a GIF with DarkMode on

Reply:
I already moved the line and I posted the GIF with DarkMode on in the PR